custom_components/ha_washdata/ml/training_task.py

```
@@ -0,0 +1,605 @@
+        if isinstance(raw_conf, (int, float)) and not isinstance(raw_conf, bool) and raw_conf > 0:
+            conf = float(raw_conf)
+            proxy_dist, proxy_margin, proxy_fit = max(0.0, 1.0 - conf), conf, conf
+        else:
+            proxy_dist, proxy_margin, proxy_fit = 0.25, 0.30, 0.75
+        try:
+            feat = quality_features(
+                points, exp["duration"], exp["energy"], exp["peak"],
+                proxy_dist, proxy_margin, proxy_fit, 0,
+            )
+        except Exception:  # pylint: disable=broad-exception-caught
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)